pkg/labels/consul_aggregator.go

```diff
@@ -13,13 +13,12 @@ import (
 )
 
 // minimum required time between retrievals of label subtrees.
-var AggregationRateCap = 10 * time.Second
+var DefaultAggregationRate = 10 * time.Second
 
 // linked list of watches with control channels
 type selectorWatch struct {
 	selector labels.Selector
 	resultCh chan *[]Labeled
-	canceled chan struct{}
```

Comment:
`canceled` is no longer necessary since failed sends on the channel due to closure will be handled gracefully.